packages/metal-quartz-button/src/MetalQuartzButtonBase.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import Component from 'metal-component';
+import core from 'metal';
+
+/**
+ * Metal Quartz Button Base component.
+ */
+class MetalQuartzButtonBase extends Component {
+}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+MetalQuartzButtonBase.STATE = {
+	/**
+	 * The button disabled attribute.
+	 * @instance
+	 * @memberof MetalQuartzButtonBase
+	 * @type {boolean}
+	 * @default false
+	 */
+	disabled: {
+		validator: core.isBoolean,
+		value: false
+	},
+
+	/**
+	 * The link href.
```

Comment:
Yeah, I think so... this was our old implementation and it had those as well: https://github.com/liferay-lexicon/lexicon-button

This should be the relevant document now: https://liferay.invisionapp.com/d/main#/console/10324888/219580168/preview